SPECS/containerized-data-importer/containerized-data-importer.spec

```diff
@@ -0,0 +1,522 @@
+#
+# If 'kubevirt_registry_path' is not specified, the standard publish location for
+# SLE and openSUSE-based containers is used.
+#
+distro='%{?sle_version}:%{?is_opensuse}%{!?is_opensuse:0}'
+case "${distro}" in
+150200:0)
+    tagprefix=suse/sles/15.2
+    labelprefix=com.suse.kubevirt
+    registry=registry.suse.com
+    ;;
+150300:0)
+    tagprefix=suse/sles/15.3
+    labelprefix=com.suse.kubevirt
+    registry=registry.suse.com
+    ;;
+150400:0)
+    tagprefix=suse/sles/15.4
+    labelprefix=com.suse.kubevirt
+    registry=registry.suse.com
+    ;;
+*)
+    tagprefix=kubevirt
+    labelprefix=org.opensuse.kubevirt
+    registry=registry.opensuse.org
+    ;;
+esac
+
+%if "%{?kubevirt_registry_path}" == ""
+    reg_path="${registry}/${tagprefix}"
+%else
+    reg_path='%{kubevirt_registry_path}'
+%endif
+
+sed -i"" \
+    -e "s#_TAGPREFIX_#${tagprefix}#g" \
+    -e "s#_LABELPREFIX_#${labelprefix}#g" \
+    -e "s#_REGISTRY_#${registry}#g" \
+    -e "s#_PKG_VERSION_#%{version}#g" \
+    -e "s#_PKG_RELEASE_#%{release}#g" \
+    %{SOURCE1}
+
+export GOPATH=%{_builddir}/go
+export GOFLAGS="-buildmode=pie -mod=vendor"
+env \
+CDI_SOURCE_DATE_EPOCH="$(date -r LICENSE +%s)" \
+CDI_GIT_COMMIT='v%{version}' \
+CDI_GIT_VERSION='v%{version}' \
+CDI_GIT_TREE_STATE="clean" \
+./hack/build/build-go.sh build \
+	cmd/cdi-apiserver \
+	cmd/cdi-cloner \
+	cmd/cdi-controller \
+	cmd/cdi-importer \
+	cmd/cdi-uploadproxy \
+	cmd/cdi-uploadserver \
+	cmd/cdi-operator \
+	%{nil}
+
+env DOCKER_PREFIX=$reg_path DOCKER_TAG=%{version}-%{release} ./hack/build/build-manifests.sh
+
+%install
+mkdir -p %{buildroot}%{_bindir}
+
+install -p -m 0755 _out/cmd/cdi-apiserver/cdi-apiserver %{buildroot}%{_bindir}/virt-cdi-apiserver
+
+install -p -m 0755 cmd/cdi-cloner/cloner_startup.sh %{buildroot}%{_bindir}/
+install -p -m 0755 _out/cmd/cdi-cloner/cdi-cloner %{buildroot}%{_bindir}/
+
+install -p -m 0755 _out/cmd/cdi-controller/cdi-controller %{buildroot}%{_bindir}/virt-cdi-controller
+
+install -p -m 0755 _out/cmd/cdi-importer/cdi-importer %{buildroot}%{_bindir}/virt-cdi-importer
+
+install -p -m 0755 _out/cmd/cdi-operator/cdi-operator %{buildroot}%{_bindir}/virt-cdi-operator
+
+install -p -m 0755 _out/cmd/cdi-uploadproxy/cdi-uploadproxy %{buildroot}%{_bindir}/virt-cdi-uploadproxy
+
+install -p -m 0755 _out/cmd/cdi-uploadserver/cdi-uploadserver %{buildroot}%{_bindir}/virt-cdi-uploadserver
+
+# Install release manifests
+mkdir -p %{buildroot}%{_datadir}/cdi/manifests/release
+install -m 0644 _out/manifests/release/cdi-operator.yaml %{buildroot}%{_datadir}/cdi/manifests/release/
+install -m 0644 _out/manifests/release/cdi-cr.yaml %{buildroot}%{_datadir}/cdi/manifests/release/
+
+# Install cdi_containers_meta build service
+mkdir -p %{buildroot}%{_prefix}/lib/obs/service
+install -m 0755 %{SOURCE1} %{buildroot}%{_prefix}/lib/obs/service
+install -m 0644 %{SOURCE2} %{buildroot}%{_prefix}/lib/obs/service
+
+# Add top-level node for building package graph
+%files
```

Comment:
Why are we building the default package, since it neither has any files nor any `Requires` to make it a metapackage? If we don't need it, we shouldn't be building it.

Reply:
As discussed, we don't need the %files. We can remove it - a full build doesn't require it.

Reply:
The empty %files node has been removed.